conf/modules.config

```diff
@@ -76,4 +76,13 @@ process {
         ext.args = '--evalue 1.0e-25  --max-target-seqs 10 --max-hsps 1'
     }
 
+    withName: 'GET_WINDOW_STATS' {
+        publishDir = [
+            path: { "${params.outdir}/blobtoolkit" },
+            mode: params.publish_dir_mode,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
+        ext.args = '--window 0.1 --window 0.01 --window 100000 --window 1000000'
```

Comment:
`ext.args` above the publishDir - just makes it easier to read and find.

Reply:
Can you double check this is working and being passed correctly? In the version I am seeing, there seems to be a missing parentheses.

Reply:
Hi there were some changes that weren't staged :D should be working now